bot/seasons/evergreen/snakes/utils.py

```diff
@@ -435,8 +436,9 @@ def startup_event_check(reaction_: Reaction, user_: Member):
                 if reaction.emoji == JOIN_EMOJI:
                     await self.player_join(user)
                 elif reaction.emoji == CANCEL_EMOJI:
-                    if self.ctx.author == user:
-                        await self.cancel_game(user)
+                    if user == self.author or not self._mod_in_game_check(user):
```

Comment:
`_mod_in_game_check()` returns `False` if the player is not in the game or if they are not a mod. Thus, this allows for anyone not in the game to cancel it.